scripts/check_frontend_test_coverage.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
Tests for this file seem to be missing, yet the backend coverage passes. This seems like a problem -- /cc @kevinlee12 @oppia/dev-workflow-team.

Reply:
This is intentional and by design. We have some python scripts like the linter that can't have tests at the moment. The gatekeeper at the moment should be the reviewer and the enforcement of 100% coverage will follow.

Reply:
@marianazangrossi even though the backend tests pass for this, there should still be tests for this file.

Reply:
Done.